specs/proposals/blob-signing.md

```diff
@@ -0,0 +1,282 @@
+    - Set up trust policy for blobs by adding a new command `notation blob policy init`. This command streamlines the process, eliminating the need for users to consult documentation for the correct trust policy format and preventing the accidental use of policies intended for other verification purposes.
+
+        ```shell
+        notation blob policy init --name "myBlobPolicy" --trust-store "ca:myCACerts" --trust-identity "x509.subject:C=US,ST=WA,O=wabbit-network.io"
+        ```
+
+        > NOTE
+        >
+        > See the [section](#manage-blob-policies) for more commands for `notation blob policy`.
+        >
+        > This `init` command can be extended to policy commands for OCI artifacts.
+    - Verify the signature:
+
+        ```shell
+        notation blob verify --policy-name myBlobPolicy --signature sbom.json.cose.sig sbom.json
+        ```
+
+        If a [global policy](https://github.com/notaryproject/specifications/blob/main/specs/trust-store-trust-policy.md#blob-trust-policy) is set, you can skip the `--policy-name` flag.
+
+### Blob Sign and verify with registry-based distribution
+
+For registry-based distribution, such as using an OCI-compliant container registry, the process is similar but includes additional steps for pushing and verifying blobs in the registry.
+
+**Prerequisites:**
+- SBOMs, release artifacts or other files are created and ready for publishing.
+
+**Steps:**
+
+1. Sign a file on filesystem with a specified signature envelope:
+
+    ```shell
+    notation blob sign sbom.json --signature-format cose --id myKeyId --plugin myKMSPluginName
+    ```
+
+    This generates a signature named `sbom.json.cose.sig` in the same directory as `sbom.json`.
+
+2. Publish the file and signature to the OCI-compliant registry:
+
+    ```shell
+    notation blob push --artifact-type "application/spdx+json" sbom.json:"application/json" --signature sbom.json.cose.sig --reference myregistry/mypath/mysboms:v1
+    ```
+
+    This command stores `sbom.json` as an OCI image referenced by `myregistry/mypath/mysboms:v1` in the registry, which has the artifact type `application/spdx+json` and `sbom.json` data stored as a layer with the media type `application/json`. Additionally, this command stores the signature file `sbom.json.cose.sig` as another OCI image with the artifact type `application/vnd.cncf.notary.signature` and the signature data as a layer with media type `application/cose`. The artifact type should be `application/vnd.cncf.notary.signature` as it is Notary Project signature. The layer media type can be detected according to the file name, which is either `application/cose` or `application/jws`. The signature manifest's `subject` property refers to the `myregistry/mypath/mysboms:v1`. To distinguish these signatures from those generated during OCI artifact signing, the specific annotation `io.cncf.notary.blob.signature=true` is added for the signature manifest.
```

Comment:
> This command stores `sbom.json` as an OCI image...

Do you mean _OCI image_ or _OCI artifact_?

Reply:
It is an OCI artifact. I want to say they are stored per the OCI image spec. I will rephase it.